Source/WebGPU/WebGPU/Queue.h

```diff
@@ -88,15 +88,18 @@ class Queue : public WGPUQueueImpl, public ThreadSafeRefCounted<Queue> {
     static bool writeWillCompletelyClear(WGPUTextureDimension, uint32_t widthForMetal, uint32_t logicalSizeWidth, uint32_t heightForMetal, uint32_t logicalSizeHeight, uint32_t depthForMetal, uint32_t logicalSizeDepthOrArrayLayers);
     void endEncoding(id<MTLCommandEncoder>, id<MTLCommandBuffer>) const;
 
+    void ensureBlitCommandEncoder();
+    void finalizeBlitCommandEncoder();
+
+    id<MTLBlitCommandEncoder> blitCommandEncoder() { RELEASE_ASSERT(m_blitCommandEncoder); return m_blitCommandEncoder; }
```

Comment:
This will release assert when the Queue is invalid and called in the below Swift extension, but maybe `blitCommandEncoder` is never called on an invalid queue?

Reply:
propose removing this `blitCommandEncoder` member function and following the pattern of CommandEncoder which has `id<MTLBlitCommandEncoder> ensureBlitCommandEncoder();`